config/initializers/devise.rb

```diff
@@ -37,7 +37,7 @@
   # ==> Configuration for :database_authenticatable
   # For bcrypt, this is the cost for hashing the password and defaults to 10. If
   # using other encryptors, it sets how many times you want the password re-encrypted.
-  config.stretches = 20
+  config.stretches = Rails.env.test? ? 1 : 20
```

Comment:
And? Did it help? Semaphore actually ran 2.5 minutes longer. But one run each is not a reliable measure.

Reply:
the build times are too unstable to see, the 5 builds before this change had times between 51 and 1h04 minutes...